inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -251,8 +254,12 @@ protected function get_field_value( array|string $field_value, string $default_v
 			case 'html':
 				return $field_value_single;
 
-			case 'price':
-				return sprintf( '$%s', number_format( (float) $field_value_single, 2 ) );
+			case 'currency':
+				$currency_symbol = $mapping['prefix'] ?? '$';
+				return sprintf( '%s%s', $currency_symbol, number_format( (float) $field_value_single, 2 ) );
+
+			case 'string_array':
+				return implode( ', ', $field_value );
```

Comment:
Wondering if this should be a separate type - `string_array` or would it be better if we keep the type as `string` and if it finds an array, it should just join them using comma.

Reply:
I think the latter is preferred. The value is a string. It's just our way of automatically mapping the array from the source. But once we join them, it's just a string and that's what it should be on our side. If you write your own config, you can use a different type and conversion.